en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,36 @@
+# SBus Driver for Linux
+This driver allows the autopilot to fetch data from the futaba sbus receiver via serial port, up to 16 channels.  
+You can also use other receivers based on the sbus protocol, such as FrSky, radioLink, and even sbus encoders.  
+Onboard serial, USB to TTL serial cable/converter, and other types of serial ports are supported.  
+All platforms running linux system are supported.  
+`This driver is located in  Firm/src/drivers/linux_sbus`
+
+## Required components
+
+1x  NPN transistor  
+1x  10K resistor  
+1x  1K  resistor  
+
+Any type of transistor can be used for this purpose. 
+
+## Optional components
+1x USB to TTL Serial Cable/Converter
```

Comment:
1. I mean "Link to a real off the shelf component that someone can buy" - unless "USB to TTL Serial Cable/Converter" is a completely unambiguous term that someone can find in any electronics store. 
2. Why is it optional? What if you don't have it? Does that change the circuit, and what does it change otherwise?